packages/game-bridge/package.json

```diff
@@ -14,8 +14,8 @@
     "parcel": "^2.8.3"
   },
   "scripts": {
-    "build": "parcel build --no-cache --no-scope-hoist && yarn updateSdkVersion",
     "d": "swc src -d dist --strip-leading-paths --ignore '**/*.test.*'",
+    "build": "parcel build --no-cache --no-scope-hoist --no-optimize  && yarn updateSdkVersion",
```

Comment:
could you help me with what does no-optimize solve?  and perhaps the double space is a mistake here?

Reply:
`--no-optimize` will stop not minify the game bridge build artefact so it'll be easier to see which bit of code triggered the error in the stack trace. Otherwise the JS output is just one giant line making it harder to parse.